doc/user-guide/Getting-started.md

```diff
@@ -2,59 +2,74 @@
 
 ### Getting started with MongooseIM on Mac OS X
 
-This short guide will show you how to compile MongooseIM from source code or install binaries from a DMG image on Mac OS X, and get users chatting right away.
+This short guide will show you how to compile MongooseIM from source code or install binaries from a DMG image on macOS, and get users chatting right away.
 
 #### Install from source code
 
 ##### Prerequisites for source code installation
 
-MongooseIM is supported on Mac OS X 10.6.8 and later. Before you can compile and run MongooseIM, you also need the following to be installed on your system:
+MongooseIM is supported on Mac OS X / macOS 10.6.8 and later.
+Before you can compile and run MongooseIM, you will need to install the following packages:
 
-* GNU Make and GCC (the GNU Compiler Collection). To ensure that these are available, you can install the Command Line Tools for Xcode, available via Xcode or from the Apple Developer website.
-* Git – `brew install git`
-* Erlang/OTP 17.5 or higher – `brew install erlang`
+* C and C++ compiler
+* Git
+* Erlang/OTP 17.5 or higher
```

Comment:
How about we suggest 18.3 or higher? MongooseIM will soon work with Erlang 18, 19 and 20. Support for 17 will be ditched after merging #1430